os/mm/umm_heap/umm_mallinfo.c

```diff
@@ -103,6 +107,14 @@ struct mallinfo mallinfo(void)
 int mallinfo(struct mallinfo *info)
 {
 	int heap_idx;
+	DEBUGASSERT(info);
```

Comment:
`mallinfo` is not an important functionality, so I think we don't need to make assert.
I think `return ERROR` is enough.